client/my-sites/purchases/subscriptions/subscriptions-content.tsx

```diff
@@ -30,7 +30,10 @@ function SubscriptionsContent( {
 	isFetchingPurchases: boolean;
 	hasLoadedPurchases: boolean;
 	selectedSiteId: number | null;
-	selectedSite: null | { ID: number; name: string; domain: string; slug: string };
+	selectedSite:
+		| undefined
+		| null
+		| { ID: number; name: string | undefined; domain: string; slug: string };
```

Comment:
Is this really supposed to be such a barebone version of the `SiteDetails`?

Reply:
Good point. This prop is only set in one place (within this same file) and it comes from `getSelectedSite()`. I'll change this to the full version.